scripting/lua/LuaReference.cpp

```diff
@@ -23,7 +23,7 @@ LuaReference::LuaReference(lua_State * L)
 	key = luaL_ref(l, LUA_REGISTRYINDEX);
 }
 
-LuaReference::LuaReference(LuaReference && other)
+LuaReference::LuaReference(LuaReference && other) noexcept
```

Comment:
Do we use `noexcept`? Is there any point in using it?
Not against this change but what are advantages of this form?

Reply:
@IvanSavenko standard library do not call move constructors if they are not marked noexcept, which leads to copy a variable everywhere. Maybe it this case this is not significant.